readme.md

```diff
@@ -724,6 +724,7 @@ The `next` API is as follows:
 
 Supported options:
 - `dev` (`bool`) whether to launch Next.js in dev mode - default `false`
+- `etag` (`bool`) whether to append etag header to HTML responses - default `true`
```

Comment:
Usually, we don't accept configuration from here. We do it via our config file `next.config.js`. Could you use that instead?

And use a verb like `disableEtags`

Reply:
We went for `generateEtags: false` @arunoda. See above